lib/sql.js

```diff
@@ -403,42 +1146,151 @@ SqlConnector.prototype.automigrate = function (models, cb) {
 };
 
 /**
- * Drop the table for the given model from the database
- * @param {String} model The model name
- * @param {Function} [cb] The callback function
+ * Serialize an object into JSON string or other primitive types so that it
+ * can be saved into a RDB column
+ * @param {Object} obj The object value
+ * @returns {*}
+ */
+SqlConnector.prototype.serializeObject = function(obj) {
+  var val;
```

Comment:
Mapping objects/array into primitive column values is the responsibility of relational DB connectors.